rest/api.go

```diff
@@ -35,10 +35,11 @@ const (
 )
 
 type rootResponse struct {
-	Admin   bool   `json:"ADMIN,omitempty"`
-	CouchDB string `json:"couchdb,omitempty"` // TODO: Lithium - remove couchdb welcome
-	Vendor  vendor `json:"vendor,omitempty"`
-	Version string `json:"version,omitempty"`
+	Admin            bool   `json:"ADMIN,omitempty"`
+	CouchDB          string `json:"couchdb,omitempty"` // TODO: Lithium - remove couchdb welcome
+	Vendor           vendor `json:"vendor,omitempty"`
+	Version          string `json:"version,omitempty"`
+	PersistentConfig bool   `json:"persistentConfig,omitempty"`
```

Comment:
We typically use `snake_case` for JSON properties in Sync Gateway.

```suggestion
	PersistentConfig bool   `json:"persistent_config,omitempty"`
```